src/js/components/Header/PlayingBar.react.js

```diff
@@ -29,7 +29,7 @@ export default class PlayingBar extends Component {
         queue: React.PropTypes.array,
         queueCursor: React.PropTypes.number,
         shuffle: React.PropTypes.bool,
-        repeat: React.PropTypes.string
+        repeat: React.PropTypes.bool
```

Comment:
Now I get it. Will fix then. How about making default case `"default"` or something?